### PR TITLE
First request

### PR DESCRIPTION
**Pull request by RafalWywrot**

### CODE REVIEW

File: Tabela.html [hunk 1]
```diff
@@ -1,42 +0,0 @@
-<!doctype html>
-<html>
-	<head>
-		<meta charset="UTF-8">
-		<title> tabela</title>
-		<link rel = "stylesheet" href = "style.css" type = "text/css" />
```
Comment (damiangolojuch) [inline Tabela.html]: Remove spaces between '='

File: Tabela.html [hunk 2]
```diff
@@ -1,42 +0,0 @@
...
-
-		<div id="tabela">
-			<table bordercolor="black" cellspacing = "0" cellpadding = "10" border = "2">
-				<tr bgcolor = "yellow" border-top-color = "blue">
-					<td> Numer  </td>  <td> Imie  </td>  <td> Nazwisko   </td>  	<td> Ulubiony Kolor  </td>  
-				</tr>
-				<tr>
-					<td> 1 </td>  	<td> Piotr </td>  	<td> Czyż </td>  	<td> Fioletowy </td>  
-				</tr>
-				<tr class = "moja">
```
Comment (damiangolojuch) [inline Tabela.html]: Remove 'class="moja"', use css selector instead.

File: style.css [hunk 3]
```diff
@@ -1,15 +0,0 @@
...
-
-td{ 
-	width: 150px; 
-	height : 25px;
-	font-size: 20px;
-	color : black;
-	text-align : center;
-}
-
-.szare_tlo"> {
```
Comment (damiangolojuch) [inline style.css]: Remove ".

File: Tabela.html (hunk 2 above)
Comment (RafalWywrot) [reply Tabela.html]: How to use css selector ?

Comment (damiangolojuch) [reply Tabela.html]: Check this article: https://www.w3schools.com/cssref/tryit.asp?filename=trycss3_nth-child_odd_even

Comment (damiangolojuch) [reply Tabela.html]: https://www.w3schools.com/cssref/sel_nth-child.asp

**Pull request merged**